<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="pt-br">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">58c11cbae0d160d2200ee4006a255a4dd2033801</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.ExecutionEngineException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">13a4fd5a96f5328869559f504e0585cd92316230</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">f56cedd9535d4cc3d9098de03bb18b9587d4568e</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown when there is an internal error in the execution engine of the common language runtime.</source>
          <target state="translated">A exceção que é lançada quando há um erro interno no mecanismo de execução do common language runtime.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve">
          <source>This class cannot be inherited.</source>
          <target state="translated">Essa classe não pode ser herdada.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT">
          <source>There is no non-obsolete alternative to ExecutionEngineException.</source>
          <target state="translated">Não há nenhuma alternativa não obsoleta para ExecutionEngineException.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT">
          <source>If further execution of your application cannot be sustained, use the &lt;xref:System.Environment.FailFast%2A&gt; method.</source>
          <target state="translated">Se ainda mais a execução do seu aplicativo não pode ser mantida, use o &lt;xref:System.Environment.FailFast%2A&gt;método.&lt;/xref:System.Environment.FailFast%2A&gt;</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ExecutionEngineException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class.</source>
          <target state="translated">Inicializa uma nova instância do <bpt id="p1">&lt;xref href="System.ExecutionEngineException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor initializes the &lt;xref:System.Exception.Message%2A&gt; property of the new instance to a system-supplied message that describes the error, such as "Internal error occurred."</source>
          <target state="translated">Este construtor inicializa a &lt;xref:System.Exception.Message%2A&gt;propriedade da nova instância para uma mensagem fornecida pelo sistema que descreve o erro, como "Erro interno."&lt;/xref:System.Exception.Message%2A&gt;</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT">
          <source>This message takes into account the current system culture.</source>
          <target state="translated">Esta mensagem leva em conta a cultura atual do sistema.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ExecutionEngineException&gt;.</source>
          <target state="translated">A tabela a seguir mostra os valores de propriedade inicial para uma instância de &lt;xref:System.ExecutionEngineException&gt;.&lt;/xref:System.ExecutionEngineException&gt;</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The localized error message string.|</source>
          <target state="translated">| Propriedade | Valor |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| Uma referência nula (<ph id="ph1">`Nothing`</ph> no Visual Basic). |   | &lt;xref:System.Exception.Message%2A&gt;| A cadeia de caracteres de mensagem de erro localizada. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ExecutionEngineException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message.</source>
          <target state="translated">Inicializa uma nova instância do <bpt id="p1">&lt;xref href="System.ExecutionEngineException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe com uma mensagem de erro especificada.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT">
          <source>This constructor initializes the &lt;xref:System.Exception.Message%2A?displayProperty=fullName&gt; property of the new exception using the <ph id="ph1">`message`</ph> parameter.</source>
          <target state="translated">Este construtor inicializa o &lt;xref:System.Exception.Message%2A?displayProperty=fullName&gt;propriedade da nova exceção usando o <ph id="ph1">`message`</ph> parâmetro.&lt;/xref:System.Exception.Message%2A?displayProperty=fullName&gt;</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT">
          <source>The content of <ph id="ph1">`message`</ph> is intended to be understood by humans.</source>
          <target state="translated">O conteúdo de <ph id="ph1">`message`</ph> se destina a ser compreendidos por humanos.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">O chamador desse construtor é necessário para garantir que essa cadeia de caracteres foi localizada para a cultura atual do sistema.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ExecutionEngineException&gt;.</source>
          <target state="translated">A tabela a seguir mostra os valores de propriedade inicial para uma instância de &lt;xref:System.ExecutionEngineException&gt;.&lt;/xref:System.ExecutionEngineException&gt;</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|A null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic).|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Propriedade | Valor |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| Uma referência nula (<ph id="ph1">`Nothing`</ph> no Visual Basic). |   | &lt;xref:System.Exception.Message%2A&gt;| A cadeia de caracteres de mensagem de erro. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>The message that describes the error.</source>
          <target state="translated">A mensagem que descreve o erro.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ExecutionEngineException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Inicializa uma nova instância do <bpt id="p1">&lt;xref href="System.ExecutionEngineException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> classe com uma mensagem de erro especificada e uma referência à exceção interna que é a causa dessa exceção.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">Uma exceção que é lançada como um resultado direto de uma exceção anterior deve incluir uma referência à exceção anterior no &lt;xref:System.Exception.InnerException%2A&gt;propriedade.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or a null reference (<ph id="ph1">`Nothing`</ph> in Visual Basic) if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">O &lt;xref:System.Exception.InnerException%2A&gt;propriedade retorna o mesmo valor que é passado para o construtor, ou uma referência nula (<ph id="ph1">`Nothing`</ph> no Visual Basic) se o &lt;xref:System.Exception.InnerException%2A&gt;propriedade não fornecer o valor da exceção interna para o construtor.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ExecutionEngineException&gt;.</source>
          <target state="translated">A tabela a seguir mostra os valores de propriedade inicial para uma instância de &lt;xref:System.ExecutionEngineException&gt;.&lt;/xref:System.ExecutionEngineException&gt;</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Propriedade | Valor |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| A referência à exceção interna. |   | &lt;xref:System.Exception.Message%2A&gt;| A cadeia de caracteres de mensagem de erro. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">A mensagem de erro que explica o motivo da exceção.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">A exceção que é a causa da exceção atual.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve">
          <source>If the <ph id="ph1">`innerException`</ph> parameter is not a null reference (<bpt id="p1">&lt;xref uid="langword_csharp_Nothing" name="Nothing" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> in Visual Basic), the current exception is raised in a <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> block that handles the inner exception.</source>
          <target state="translated">Se o <ph id="ph1">`innerException`</ph> parâmetro não é uma referência nula (<bpt id="p1">&lt;xref uid="langword_csharp_Nothing" name="Nothing" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> no Visual Basic), a exceção atual é gerada em um <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt> <ept id="p2">&lt;/xref&gt;</ept> bloco que manipula a exceção interna.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>